RxBlocking.podspec

```diff
@@ -26,5 +26,5 @@ Waiting for observable sequence to complete before exiting command line applicat
   s.exclude_files         = 'RxBlocking/Platform/**/*.swift'
 
   s.dependency 'RxSwift', '~> 5'
-  s.swift_version = '5.0'
+  s.swift_version = '5.1'
```

Comment:
I'm not sure it's needed

Reply:
How come? We're targeting Swift 5.1 and up.

Reply:
My understanding is that this value is going to set the equivalent SWIFT_VERSION as build setting. There is no 5.1 in Xcode build settings. Apparently, if your project is compatible with swift 5, and you are running xcode11, then you are on swift 5.1;

From cocoa pods documentation:
> Note The Swift compiler mostly accepts major versions and sometimes will honor minor versions. While CocoaPods allows specifying a minor or patch version it might not be honored fully by the Swift compiler.

So I would say that I might be wrong and it's ok to put 5.1, even if Xcode is going to ignore it and mark it simply as swift 5... I wonder if there is a way to constrain to Xcode 11.... doing some research.

Reply:
didn't find a way to constrain Xcode version... I'd say that 5.1 is probably fine.

Reply:
I searched, :D I swear I did, but I didn't find a way to impose xcode11 specifically. (which I suppose it also makes sense). I would say that we can keep 5.1 in the spec, Xcode will probably honour 5.0 requirement, but at the very least the spec is clear about requirements.